projects/ngx-oxford/src/lib/oxford.pipe.ts

```diff
@@ -17,7 +27,14 @@ export class OxfordPipe implements PipeTransform {
       throw new TypeError(`Wrong conjunction for Oxford Pipe provided: ${conjunction} is not a word`);
     }
 
-    const trimmedValues = values.map(OxfordPipe.trim);
+    let trimmedValues = values.map(OxfordPipe.trim);
+
+    if (truncate) {
+      if (truncate.wordCount > values.length) {
```

Comment:
If the item count is less than the truncation count, it should just proceed as usual.

I think throwing is unexpected, and removes some of the usefulness since there would be no use to the conjunction. And the code using it will have to know ahead of time or verify the length of the list.

So with "and" and truncate at 3 items:
```
a, b, c     =>  a, b, and c
a, b, c, d  =>  a, b, c, ...
```

Reply:
Done